dev/Repeater/BuildTreeScheduler.h

```diff
@@ -10,7 +10,7 @@ struct WorkInfo
         m_workFunc(workFunc)
     {}
 
-    int Priority() const { return m_priority; }
+    [[nodiscard]] int Priority() const { return m_priority; }
     void InvokeWorkFunc() const { m_workFunc(); }
```

Comment:
Hmm ... I like [[nodiscard]] when the type being returned is an error code or a struct with non-trivial destructor, but I'm not liking the places where it's being added in this change. Can we not add [[nodiscard]] in this change and leave it for a separate cleanup?

Reply:
sounds good.